orttraining/orttraining/test/training_ops/cuda/resize_grad_test.cc

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "test/providers/compare_provider_test_utils.h"
+#include "test/providers/provider_test_utils.h"
+#include "test/util/include/default_providers.h"
+
+namespace onnxruntime::test {
+
+#ifdef USE_CUDA
```

Comment:
also allow USE_ROCM?

Reply:
I added `USE_ROCM` for this file.
Unfortunately, I am not able to run the test when I add the macro to the gradient checker test. It complains about not having an implementation for `ResizeGrad`. I wonder if it is complaining about the CPU kernel for some reason.
I left that as a todo for now. Let me know if you have any ideas around why it might be complaining.